cogs/Verification.py

```diff
@@ -443,5 +444,5 @@ def setup(bot: KoalaBot) -> None:
         KoalaBot.database_manager.insert_extension("Verify", 0, False, False)
     else:
         bot.add_cog(Verification(bot))
-        print("Verification is ready.")
-
+        logging.info("Verification is ready.")
+        print("Verification is ready.")
```

Comment:
Is it necessary to print and log it? couldn't it just be logged at a level that is already printed to console automatically

Reply:
logging currently is poor, but is improved in #251 and can be replaced later